app/views/accounts/_pii_expired.html.slim

```diff
@@ -0,0 +1,46 @@
+.mb3.profile-info-box
```

Comment:
I think making a separate template like this makes it super easy to drift away from the `_pii.html.slim` template. I think we could pass in a `decrypted_pii`-like object that just responds with redacted values instead?